docs/custom_resource.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Custom Resource Definitions"
+description: "Configure your custom resources so that Tilt can monitor them."
+layout: docs
+---
+
+Kubernetes defines well-thought-out primitives like Deployments, Jobs, and
+StatefulSets for running your containers.
+
+Tilt knows how to inject images, monitor status, and fetch logs for these
+built-in resources.
+
+But your team may define your own custom resources! Or you may use a project
+like KubeDB that uses custom resources to operate a database.
+
+Tilt needs your help to recognize them.
+
+## Custom Resource Functions
+
+For any resource, Tilt needs to know 3 things:
+
+### Is this an indepdent resource?
```

Comment:
```suggestion
### Is this an independent resource?
```

But also, what happens if it's not an independent resource, what should you do then?

(I know the answer, I just think that the way that this is phrased leaves that question unanswered)

Reply:
done